mhkit/tests/test_loads.py

```diff
@@ -41,6 +48,45 @@ def test_bin_statistics(self):
         assert_frame_equal(self.data['bin_means'],b_means)
         assert_frame_equal(self.data['bin_means_std'],b_means_std)
 
+
+    def test_calculate_TSR(self):
+        rotor_speed = [15,16,17,18] # create array of rotor speeds
+        rotor_diameter = 77 # diameter of rotor for GE 1.5
+        inflow_speed = [13,13,13,13] # array of wind speeds
+        TSR_answer = [4.7,5.0,5.3,5.6]
+        
+        TSR = loads.calculate_TSR(rotor_speed,rotor_diameter,inflow_speed)
+        error = np.abs((TSR_answer - TSR)/TSR_answer)
+
+        self.assertTrue((error < 0.015).all())
```

Comment:
Will you consider using the built-in function [assertAlmostEqual](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual) for this one and where appropriate below?

Reply:
updated the tests to use assertAlmostEqual